src/components/styles/NavBar.styled.ts

```diff
@@ -4,5 +4,10 @@ export const MainContainer = styled.div`
   width: 65%;
   display: flex;
   margin-left: 10%;
-  justify-content: space-evenly;
+  justify-content: space-between;
+`
+
+export const NavContainer = styled.div`
+  width: 100%;
```

Comment:
might want to use 100vw instead? not sure if this might cause issues

Reply:
i was under the impression they did the same things, but will change